packages/web/src/pages/sign-in-page/SignInPage.tsx

```diff
@@ -85,7 +85,6 @@ export const SignInPage = () => {
       const { email, password } = values
       dispatch(setValueField('email', email))
       dispatch(setValueField('password', password))
-      dispatch(signIn(email, password))
```

Comment:
moving the signin call to the confirm email page above. seems better to consolidate it there because there's multiple places that link to it